tigerpath/majors_and_certificates/majors/REL_2018.json

```diff
@@ -0,0 +1,221 @@
+  "urls": [
+    "https://ua.princeton.edu/academic-units/department-religion",
+    "https://religion.princeton.edu/undergraduate-program/major-requirements/"
+  ],
+  "contacts": [
+    {
+      "type": "Chair",
+      "name": "Judith L. Weisenfeld",
+      "email": "jweisenf@princeton.edu"
+    },
+    {
+      "type": "Director of Undergraduate Studies",
+      "name": "Seth Perry",
+      "email": "sethp@princeton.edu"
+    }
+  ],
+  "req_list": [
+    {
+      "name": "Prerequisites",
+      "max_counted": 1,
+      "min_needed": 1,
+      "explanation": "Any course offered by the department.",
+      "double_counting_allowed": true,
+      "course_list": [
+        "REL *"
+      ]
+    },
+    {
+      "name": "Subfields",
+      "max_counted": 1,
+      "min_needed": "ALL",
+      "explanation": "Students will select at least one course from each of the five subfields of the faculty: Islam, Religion and Critical Thought, Religion in America, Religions of Asia, Religions of the Ancient Mediterranean.",
+      "req_list": [
+        {
+          "name": "Islam",
+          "max_counted": 1,
+          "min_needed": 1,
+          "explanation": null,
+          "course_list": [
+            "REL 235/NES 235",
+            "REL 239/NES 239",
+            "REL 327/GSS 298",
+            "REL 328/GSS 328",
+            "REL 333/NES 333",
+            "REL 335/NES 356",
+            "NES 240/REL 240"
+          ]
+        },
+        {
+          "name": "Religion and Critical Thought",
+          "max_counted": 1,
+          "min_needed": 1,
+          "explanation": null,
+          "course_list": [
+            "REL 215",
+            "REL 260/GHP 270",
+            "REL 261/CHV 261",
+            "REL 263",
+            "REL 264/CHV 264/PHI 264",
+            "REL 292",
+            "REL 293",
+            "REL 306",
+            "REL 311",
+            "REL 330/HUM 330/JDS 331/COM 382",
+            "REL 347/JDS 347",
+            "REL 364/HUM 364/GSS 412",
+            "REL 394",
+            "REL 398/JDS 398",
+            "REL 402/PHI 402/CHV 407",
+            "REL 420"
+          ]
+        },
+        {
+          "name": "Religion in America",
+          "max_counted": 1,
+          "min_needed": 1,
+          "explanation": null,
+          "course_list": [
+            "REL 250/AAS 250",
+            "REL 256/AAS 256",
+            "REL 259/GSS 278/LAS 259",
+            "REL 271/AMS 341",
+            "REL 275/LAS 275",
+            "REL 357/HIS 310",
+            "REL 359/LAS 388",
+            "REL 360/GSS 360/AMS 369",
+            "REL 371",
+            "REL 373",

```

Comment:
The department says that this course might count for a different area requirement depending on the semester and the subject covered in that semester.
Because of this, I think it should probably appear either under all the areas (letting the user decide where to settle it) or none of them (essentially indicating they shouldn't assume that it will count anywhere).
Either way, we could also add the note from the [department](https://religion.princeton.edu/undergraduate-program/major-requirements/) about this course to the end of the "Subfields" explanation.

Reply:
I got the subfield course lists from the department's past semester course lists, which includes the subfields the courses fall under for that semester. So far REL 373 has only been categorized as part of the Religion in America subfield. I think to keep TigerPath accurate, it can be added to the other subfields when there is a semester where it's categorized in another subfield.

Reply:
Fair enough. We just need to keep an eye out for it then, I guess.